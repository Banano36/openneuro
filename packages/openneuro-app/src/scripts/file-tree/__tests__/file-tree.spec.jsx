import React from 'react'
import { shallow } from 'enzyme'
import FileTree from '../file-tree.jsx'

describe('FileTree component', () => {
  it('renders with default props', () => {
    expect(shallow(<FileTree />)).toMatchSnapshot()
  })
})
