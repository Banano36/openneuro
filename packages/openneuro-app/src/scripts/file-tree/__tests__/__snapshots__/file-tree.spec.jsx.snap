// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FileTree component renders with default props 1`] = `
<FileTree>
  <Styled(div)>
    <div
      className="css-1b7l7mx"
    >
      <button
        className="btn-file-folder"
        onClick={[Function]}
      >
        <i
          className="type-icon fa fa-folder"
        />
         
        <Styled(span)>
          <span
            className="css-9ywbij"
          />
        </Styled(span)>
        <i
          className="accordion-icon fa fa-caret-down"
        />
      </button>
      <Styled(div)
        style={
          Object {
            "height": undefined,
            "opacity": undefined,
          }
        }
      >
        <div
          className="css-fcnj91"
          style={
            Object {
              "height": undefined,
              "opacity": undefined,
            }
          }
        >
          <div
            data-testid="ANIMATED-COMPONENT"
          />
        </div>
      </Styled(div)>
    </div>
  </Styled(div)>
</FileTree>
`;
